packages/node/test/e2e/cli-generate.spec.js

```diff
@@ -123,14 +101,15 @@ test('extensionless bin script handling', async (t) => {
   }
 })
 
-test('generate - policy generation - canonical names', async (t) => {
+test('canonical names', async (t) => {
```

Comment:
I would like to encourage test titles to be long and verbose. 

[group] - [thing being tested] - [what aspect of it is being tested / what outcome is being ensured]
// maybe a comment saying why it matters

I know I'm being a hypocrite and I've failed to do it myself - but I've recently experienced how much of a difference a verbose test description makes in setting me up to understand the testcase on first read or find the one that I want.

Reply:
I would prefer deeply-nested suites and verbose test titles, but alas.

AVA prefixes test titles by dir and test file basename in its reporting. So everything in here will begin with `e2e - cli-generate - `.  In this way, it seems AVA incentivizes users to define "test suites" via the filesystem structure; to "group" things you would instead move the test to a new directory and/or file.

Anyway, sure, I'll write a better title.